packaging/dpkg/docker/debian-12/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM debian:bookworm
+
+# Install dependencies
+RUN apt-get update && apt-get install -y \
+    php-dev python3 python3-dev python3-passlib ruby-full gdb \
+    libbluetooth-dev libbz2-dev libdbus-1-dev libedit-dev libexpat1-dev liblmdb-dev \
+    libmcpp-dev libssl-dev libsystemd-dev \
+    dh-exec dh-php dh-python locales-all php-all-dev python3-setuptools \
+    build-essential \
+    && rm -rf /var/lib/apt/lists/* \
+    && apt-get clean
```

Comment:
I would be best to install build dependencies dynamically when building the package, I leave that for a follow-up PR.

This will ensure that the package build dependencies are correct.